src/raftstore/store/peer.rs

```diff
@@ -957,8 +957,8 @@ impl Peer {
         }
         self.mut_store().apply_state = res.apply_state.clone();
         self.mut_store().applied_index_term = res.applied_index_term;
-        self.peer_stat.written_keys += res.metrics.written_keys;
-        self.peer_stat.written_bytes += res.metrics.written_bytes;
+        self.peer_stat.total_written_keys += res.metrics.written_keys;
```

Comment:
Do we need to update `peer_stat` for followers and learners? And, I think we also need to clear `peer_stat` in `on_role_changed` is we do so.

Reply:
If we change `peer_stat`'s behavior, it may incompatible with PD and it becomes a breaking change.